components/script_traits/lib.rs

```diff
@@ -399,8 +399,12 @@ pub enum MozBrowserEvent {
     LoadStart,
     /// Sent when a browser `<iframe>`'s location changes.
     LocationChange(String, bool, bool),
-    /// Sent when window.open() is called within a browser `<iframe>`.
-    OpenWindow,
+    /// Sent when a new tab is opened within a browser `<iframe>`.
+    /// Includes the URL and the target browsing context name.
```

Comment:
MDN doc: "The mozbrowseropentab event is fired when a new tab is opened within a browser <iframe> as a result of the user issuing a command to open a link target in a new tab (for example ctrl/cmd + click.)"